platforms/php/composer/Dockerfile

```diff
@@ -3,10 +3,14 @@ FROM mcr.microsoft.com/mirror/docker/library/buildpack-deps:${DEBIAN_FLAVOR} AS
 ARG DEBIAN_FLAVOR
 ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR
 COPY platforms/php/prereqs /php
-COPY build/__phpVersions.sh /php/
+# COPY build/__phpVersions.sh /php/
 COPY platforms/php/prereqs/build.sh /tmp/
 COPY images/receiveGpgKeys.sh /tmp/receiveGpgKeys.sh
 
+ARG PHP81_VERSION='8.1.29'
```

Comment:
same.

Reply:
As of now, we are doing mainly for runtime images, we will modify these in next pr